packages/common/src/chains/mainnet.json

```diff
@@ -77,32 +77,60 @@
   ],
   "bootstrapNodes": [
     {
-      "ip": "13.93.211.84",
-      "port": 30303,
-      "id": "3f1d12044546b76342d59d4a05532c14b85aa669704bfe1f864fe079415aa2c02d743e03218e57a33fb94523adb54032871a6c51b2cc5514cb7c7e35b3ed0a99",
-      "location": "US-WEST",
-      "comment": "Go Bootnode"
+      "ip": "18.138.108.67",
+      "port": "30303",
```

Comment:
To change the type for `port` would be a breaking change for the library, please don't side-introduce here.

Reply:
good point as this is not covered by any test 👍

Reply:
actually it kind of was and I "fixed" it :D

Reply:
😛